Makefile

```diff
@@ -1,24 +1,22 @@
 NXDK_DIR = $(CURDIR)/../nxdk
 NXDK_NET = y
 
-XBE_TITLE = triangle
+XBE_TITLE = nxdk-rdt
 GEN_XISO = $(XBE_TITLE).iso
 SRCS = $(wildcard $(CURDIR)/*.c)
 SHADER_OBJS = ps.inl vs.inl
 
 SRCS += $(CURDIR)/lib/protobuf-c/protobuf-c.c
 
-CFLAGS_EXTRA += -I$(CURDIR)
-CFLAGS_EXTRA += -I$(CURDIR)/lib
+CFLAGS += -I$(CURDIR)
+CFLAGS += -I$(CURDIR)/lib
```

Comment:
In my local copy of NXDK I actually changed the Makefile to allow you to define `CFLAGS_EXTRA` which would be appended to `CFLAGS`. If you just want to use `CFLAGS` here instead, that should be okay, but you would need to put it after the `include` statement on line 14, because the top level Makefile will re-define it and blow away what you put in `CFLAGS`.